docs/user/openstack/README.md

```diff
@@ -45,7 +45,7 @@ enough to store the ignition config files, so they are served by swift instead.
   values. For example (as an OpenStack admin) `openstack quota set --secgroups 100 --secgroup-rules 1000 <project>`
 
 * The installer requires a proper RHCOS image in the OpenStack cluster or project:
-`openstack image create --container-format=bare --disk-format=qcow2 --file redhat-coreos-${RHCOSVERSION}-openstack.qcow2 redhat-coreos-${RHCOSVERSION}`
+`openstack image create --container-format=bare --disk-format=qcow2 --file redhat-enterprise-linux-coreos-${RHCOSVERSION}-openstack.qcow2 redhat-enterprise-linux-coreos-${RHCOSVERSION}`
```

Comment:
Do we want to move to `red-hat` while we're at it?

Reply:
Or use `rhcos-${RHCOSVERSION}`?

Reply:
/cc @mrguitar :arrow_up:

Reply:
I think something akin to RHCOS-${version}-${platform}.type be easier. RHEL follows a similar pattern today: rhel-server-7.5-x86_64-dvd.iso. Expanding red-hat-enterprise-linux seems cumbersome to me.

Reply:
updating